crates/core/component/auction/src/component/dutch_auction.rs

```diff
@@ -459,6 +459,7 @@ pub(crate) trait DutchAuctionData: StateRead {
             .boxed()
     }
 
+    #[allow(dead_code)]
     async fn stream_dutch_state_by_trigger(
```

Comment:
in favor of removing it entirely

Reply:
good to know! i'll circle back to this when i have a moment

Reply:
removed in the (squashed, rebased) fa1d7b0